src/main/java/com/name/OX/Judge.java

```diff
@@ -0,0 +1,52 @@
+package com.name.OX;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * @author Ola Podorska
+ */
+//TODO: implement
+class Judge implements Observer {
+
+    private List<Move> crossMoves = new ArrayList<>();
+    private List<Move> naughtMoves = new ArrayList<>();
+    private Checker checker;
+    private GameOptions gameOptions;
+    private Symbol winner = null;
+
+    Judge(GameOptions gameOptions) {
+        this.gameOptions = gameOptions;
+    }
+
+    Symbol foundWinner() {
+        return winner;
+    }
+
+    @Override
```

Comment:
W tym ifie i elsie dzieje się praktycznie to samo, różnica jest tylklo między CROSS i NAUGHT. Może by to przenieść do osobnej metodki, do której przekażemy ruch kółka, albo krzyżyka? ;)